### DISCUSSION TITLE
Climbing boots g work for master step

### DISCUSSION THREAD

**Discussion by GamesGotIssues** [runelite/runelite#16003] [Q&A] [answered]
For the master step in the watchtower i don't have the stash built and i use d chain g and d skirt g which turns the plugin green that i have it in my inventory but it does not do the same for climbing boots g. I'm dumb and i run to get new ones every time. can it just be updated that its an accepted item?

**Comment by Nightfirecat** [comment, marked as answer]
I've opened #16004 for tracking this.
